python/tk_multi_publish2/dialog.py

```diff
@@ -668,9 +668,36 @@ def _create_item_details(self, tree_item):
             self.ui.context_widget.show()
 
             if item.context_change_allowed:
-                self.ui.context_widget.enable_editing(
-                    True, "<p>Task and Entity Link to apply to the selected item:</p>"
-                )
+                # Check for task_required and change UI styling accordingly
+                if not self._validate_task_required():
+                    # Change task label color to SG_ALERT_COLOR
+                    self.ui.context_widget.ui.task_label.setStyleSheet(
```

Comment:
Like I mentioned bellow, I'm not sure it's a good idea to go via the `context_widget.ui`, the widgets in there might get updated. Though it looks like this particular label is not exposed publicly.

I guess the correct thing to do would be to update the qtwidgets framework to publicly expose that label.
@jfboismenu what do you think?